### ISSUE TITLE
Log Error

### ISSUE THREAD

**Issue by t9001** [twrecked/hass-aarlo#321]
Not sure it's a real problem, but it asks me to report it to "the custom component author" so here goes...

```
Logger: homeassistant.util.async_
Source: util/async_.py:105 
First occurred: 20:39:39 (1 occurrences) 
Last logged: 20:39:39

Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for aarlo doing I/O at custom_components/aarlo/pyaarlo/backend.py, line 79: r = self._session.post(url, json=params, headers=headers, timeout=timeout)
```

R Pi, latest Hassio, latest Aarlo

Thanks for the great work!

**Comment by twrecked** [comment]
Are you using any of the `camera.aaarlo_*` services? Start recording or whatever?

**Comment by t9001** [comment]
This was on a restart, sorry, should have mentioned that. Any other info I can try to get that might help?

**Comment by twrecked** [comment]
I'll give it a go here. I know calling the deprecated services can give rise to that message.

**Comment by t9001** [comment]
Then it could be my bad, I have an arlo mode set on startup. Let me check to see if that's using a deprecated service

**Comment by t9001** [comment]
I _think_ this code is valid, it's the only aarlo code triggered on startup:

```
- service: aarlo.alarm_set_mode #alarm_control_panel.aarlo_set_mode
      data_template:
        entity_id: alarm_control_panel.aarlo_base
        mode: >
          {% if states('input_select.global_camera_mode') == "All On" %}
            {% if states('sensor.dark_sky_sensor_wind_speed') | int >= 10 %}
              All On Very Windy
            {% elif states('sensor.dark_sky_sensor_wind_speed') | int >= 5 %}
              All On Windy
            {% elif states('sensor.dark_sky_sensor_wind_speed') | int >= 2 %}
              All On Normal
            {% else %}
              All On Calm
            {% endif %}
          {% else %}
            {{states('input_select.global_camera_mode')}}
          {% endif %}`
```

**Comment by twrecked** [comment]
That's definitely causing the message. I'll fix it.

**Comment by ynazar1** [comment]
This seems to happen on any mode change for me WHEN using automation
`2020-10-07 16:32:04 WARNING (MainThread) [homeassistant.util.async_] Detected I/O inside the event loop. This is causing stability issues. Please report issue to the custom component author for aarlo doing I/O at custom_components/aarlo/pyaarlo/backend.py, line 79: r = self._session.post(url, json=params, headers=headers, timeout=timeout)
2020-10-07 16:32:05 INFO (MainThread) [custom_components.aarlo.alarm_control_panel] alarm_control_panel.aarlo_base_arlo setting mode to (armed/)disarmed` 

My automation example: 
`pi@scorpi:~/.homeassistant/automations $ cat ./arlo_on.yaml
alias: Arlo On from boolean
initial_state: 'on'
trigger:
  - platform: state
    entity_id: input_boolean.arlo_state
    to: 'on'
action:
  - service: aarlo.alarm_set_mode
    data:
      entity_id: alarm_control_panel.aarlo_base_arlo
      mode: 'armed'
`

**Comment by twrecked** [comment]
I'm testing a fix. A fix that also tidies up the code...

**Comment by twrecked** [comment]
Should be fixed with #324 ; can you try 0.7.0.beta.6.1.

**Comment by ynazar1** [comment]
Log below. I think, looks good.
'''
2020-10-07 21:04:17 INFO (MainThread) [homeassistant.components.automation.arlo_off_from_boolean] Arlo Off from boolean: Running automation actions
2020-10-07 21:04:17 INFO (MainThread) [homeassistant.components.automation.arlo_off_from_boolean] Arlo Off from boolean: Executing step call service
2020-10-07 21:04:17 INFO (SyncWorker_44) [custom_components.aarlo.alarm_control_panel] alarm_set_mode service called
2020-10-07 21:04:18 INFO (SyncWorker_44) [custom_components.aarlo.alarm_control_panel] alarm_control_panel.aarlo_home_arlo setting mode to disarmed
2020-10-07 21:04:34 INFO (MainThread) [homeassistant.components.automation.arlo_on_from_boolean] Arlo On from boolean: Running automation actions
2020-10-07 21:04:34 INFO (MainThread) [homeassistant.components.automation.arlo_on_from_boolean] Arlo On from boolean: Executing step call service
2020-10-07 21:04:34 INFO (SyncWorker_17) [custom_components.aarlo.alarm_control_panel] alarm_set_mode service called
2020-10-07 21:04:35 INFO (SyncWorker_17) [custom_components.aarlo.alarm_control_panel] alarm_control_panel.aarlo_home_arlo setting mode to armed
'''
